.github/workflows/ci.yml

```diff
@@ -240,6 +240,18 @@ jobs:
     - run: echo -e '#include "stim.h"\nint main(int argc,const char **argv) {return !stim::find_bool_argument("test", argc, argv);}' > test.cc
     - run: g++ -std=c++17 test.cc out/libstim.a -I src
     - run: ./a.out test
+  build_lib_install:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: mkdir install_dir
```

Comment:
Add a documentation version of this check's contents to doc/developer_documentation.md